charts/linkerd2/templates/web-rbac.yaml

```diff
@@ -5,6 +5,82 @@
 ###
 ---
 {{- if not .RestrictDashboardPrivileges }}
```

Comment:
Should we change the description for the flag that controls this option? Right now it reads:

```
      --restrict-dashboard-privileges             Restrict the Linkerd Dashboard's default privileges to disallow Tap
```

Maybe we should rephrase to "... disallow Tap and Check" or something like that.

Reply:
Done, thanks!